cpp/unreal_projects/SpearSim/SpearSim.uproject

```diff
@@ -1,44 +1,40 @@
 {
-    "FileVersion": 3,
-    "EngineAssociation": "5.2",
-    "Category": "",
-    "Description": "",
-    "Modules": [
-        {
-            "Name": "SpearSim",
-            "Type": "Runtime",
-            "LoadingPhase": "Default"
-        },
-        {
-            "Name": "SpearSimEditor",
-            "Type": "Editor",
-            "LoadingPhase": "Default"
-        }
-    ],
-    "Plugins": [
-        {
-            "Name": "CommonModuleRules",
-            "Enabled": true
-        },
-        {
-            "Name": "CoreUtils",
-            "Enabled": true
-        },
-        {
-            "Name": "HDRIBackdrop",
-            "Enabled": true
-        },
-        {
-            "Name": "SimulationController",
-            "Enabled": true
-        },
-        {
-            "Name": "UrdfBot",
-            "Enabled": false
-        },
-        {
-            "Name": "Vehicle",
-            "Enabled": true
-        }
-    ]
-}
+	"FileVersion": 3,
```

Comment:
I don't want these changes. The main difference to our existing code is that you're using tabs instead of spaces. But our convention is to use spaces. I have commented on this exact issue in our previous meeting. You're also disabling the `SimulationController` which breaks our existing code. Why are you doing this? Please revert this entire file.